pan_firewall/assets/logs/pan.firewall.yaml

```
@@ -230,4 +385,1028 @@ pipeline:
+          enabled: true
+          sources:
+            - proto
+          sourceType: attribute
+          target: ocsf.connection_info.protocol_name
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'bytes' to 'ocsf.traffic.bytes'
+          enabled: true
+          sources:
+            - bytes
+          sourceType: attribute
+          target: ocsf.traffic.bytes
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'bytes_out' to 'ocsf.traffic.bytes_out'
+          enabled: true
+          sources:
+            - bytes_out
+          sourceType: attribute
+          target: ocsf.traffic.bytes_out
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'bytes_in' to 'ocsf.traffic.bytes_in'
+          enabled: true
+          sources:
+            - bytes_in
+          sourceType: attribute
+          target: ocsf.traffic.bytes_in
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'pkts' to 'ocsf.traffic.packets'
+          enabled: true
+          sources:
+            - pkts
+          sourceType: attribute
+          target: ocsf.traffic.packets
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'pkts_sent' to 'ocsf.traffic.packets_out'
+          enabled: true
+          sources:
+            - pkts_sent
+          sourceType: attribute
+          target: ocsf.traffic.packets_out
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'pkts_received' to 'ocsf.traffic.packets_in'
+          enabled: true
+          sources:
+            - pkts_received
+          sourceType: attribute
+          target: ocsf.traffic.packets_in
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'chunks' to 'ocsf.traffic.chunks'
+          enabled: true
+          sources:
+            - chunks
+          sourceType: attribute
+          target: ocsf.traffic.chunks
+          targetType: attribute
+          targetFormat: integer
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'chunks_sent' to 'ocsf.traffic.chunks_out'
```

```suggestion
          name: Map `chunks_sent` to `ocsf.traffic.chunks_out`
```

To save me going through all of these, if you wouldn't mind converting them to 

```
Map `src` to `dst`